go-server-server/go/default.go

```diff
@@ -1075,6 +1075,20 @@ func ConfigVrouterVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
     defer local_pt.Delete()
 
     for _, r := range attr {
+
+        ip, network, err := net.ParseCIDR(r.IPPrefix)
```

Comment:
Change looks good to me. It is better to provide some comments above and an example on expectation for future clarity.